libparsec/crates/platform_mountpoint/src/lib.rs

```diff
@@ -9,8 +9,10 @@ mod windows;
 
 use chrono::{DateTime, Utc};
 use std::{collections::HashMap, path::Path};
+// TODO: remove me once winify is used
+#[allow(unused_imports)]
```

Comment:
Since you re-export winify fn the `allow(unused_imports)` could be removed

Reply:
Wait I will use winify for windows